src/smt/command.cpp

```diff
@@ -1641,15 +1640,14 @@ void GetAssignmentCommand::invoke(api::Solver* solver, SymbolManager* sm)
     // of whether terms is empty.
     std::vector<api::Term> values = solver->getValue(terms);
     Assert(values.size() == names.size());
-    std::vector<SExpr> sexprs;
+    std::vector<api::Term> sexprs;
     for (size_t i = 0, nterms = terms.size(); i < nterms; i++)
     {
-      std::vector<SExpr> ss;
-      ss.emplace_back(SExpr::Keyword(names[i]));
-      ss.emplace_back(SExpr::Keyword(values[i].toString()));
-      sexprs.emplace_back(ss);
+      // Put the name of the expression inside of a variable name for printing
+      api::Term name = solver->mkVar(solver->getBooleanSort(), names[i]);
```

Comment:
I assume its wrong to use a string constant here?  Why is that?

Reply:
Modified the comment. Please take another look.